overlord/configstate/configcore/kernel_test.go

```diff
@@ -421,12 +441,13 @@ func (s *kernelSuite) TestConfigureKernelCmdlineConflict(c *C) {
 
 	rt.Set("core", "system.kernel.dangerous-cmdline-append", cmdline)
 
-	err := configcore.Run(coreDev, rt)
+	err := configcore.Run(core20Dev, rt)
 	c.Assert(err, ErrorMatches, "kernel command line already being updated, no additional changes for it allowed meanwhile")
 }
 
 func (s *kernelSuite) TestConfigureKernelCmdlineSignedGradeNotAllowed(c *C) {
-	s.mockModel("signed")
+	isClassic := false
+	s.mockModelWithModeenv("signed", isClassic)
 	s.mockGadget(c)
```

Comment:
shouldn't this test use core20Dev too for consistency?

Reply:
Good catch, changed